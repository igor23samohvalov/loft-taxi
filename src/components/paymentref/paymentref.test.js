import React from 'react';
import { render } from '@testing-library/react';
import Paymentref from './paymentref.js';

describe('paymentref', () => {
    it('renders correctly', () => {
        const { container } = render(<Paymentref />)
        expect(container).toMatchSnapshot()
    });
});