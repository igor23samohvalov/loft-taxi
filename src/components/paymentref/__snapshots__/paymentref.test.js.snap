// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paymentref renders correctly 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
    style="position: absolute; top: 120px; left: 20px; padding: 44px 60px; max-width: 30%; flex-wrap: wrap;"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-justify-xs-space-between"
      style="height: 200px;"
    >
      <h4
        class="MuiTypography-root MuiTypography-h4"
      >
        Заполните платежные данные
      </h4>
      <p>
        Укажите информацию о банковской карте, чтобы сделать заказ.
      </p>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        data-id="Профиль"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Перейти в профиль
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
